### ISSUE TITLE
Message ids are inconsistently spelled: 21 of 315 use an underscore

### ISSUE THREAD

**Issue by raeq** [w3c/epubcheck#1728]
In `MessageId.java`, 21 of the 315 ids carry an underscore in the string epubcheck reports, where
every other id carries a hyphen:

```java
MED_005("MED-005"),
MED_006("MED_006"),
```

The 21 in 5.3.0:

```
HTM_053 HTM_054 HTM_055 HTM_056 HTM_057 HTM_058 HTM_059 HTM_060a HTM_060b HTM_061
MED_006 MED_007 MED_010 MED_011 MED_012 MED_013 MED_014 MED_015 MED_016 MED_017 MED_018
```

In 5.4.0 there are 23 of 323, with `HTM_062` and `HTM_063` added.

## Effect

One report can carry both spellings:

```
ERROR    HTM_056    Viewport metadata has no "height" dimension (both "width" and "height" ...
ERROR    RSC-012    Fragment identifier is not defined.
```

A consumer that matches ids as `[A-Z]{3}-[0-9]+` drops those 21 messages and says nothing. I hit
this while replaying epubcheck's own test corpus against two validators.

The feature files write the hyphen form for the same ids, so an expectation and the id it expects
are spelled differently:

```
src/test/resources/epub3/09-media-overlays/media-overlays.feature:98
  Then error MED-014 is reported
src/test/resources/epub3/08-layout/layout.feature:253
  Then error HTM-056 is reported
```

## Reproducing

`repro.sh` in https://gist.github.com/raeq/d617a4777361f9f5c906445d19a5c66e builds a minimal fixed-layout EPUB with two faults
and prints the ids epubcheck reports. Tested with 5.3.0 (Homebrew) on OpenJDK 26.0.2.

## Suggested fix

Spell the 21 with hyphens in the enum. `MessageId.fromString` looks ids up in a map built from the
same strings, so adding the old spellings to that map keeps a tool that matches on `HTM_056`
working.

**Comment by rdeltour** [comment]
Thanks for the report, good catch.

The ID strings should be generated from the enum key, that would make sure this does not happen again in the future.

**Comment by raeq** [comment]
@rdeltour would you like me to submit a PR for this fix?

**Comment by rdeltour** [comment]
Thanks for the offer @raeq but no need for a PR.
I will include this after v5.4.1, in the following regular maintenance release.
